src/App.tsx

```diff
@@ -4,17 +4,19 @@ import PostsWrap from './components/PostsWrap';
 import { initiatePostHog } from './lib/analytics';
 import { useState } from 'react';
 import RepoSubmission from './components/RepoSubmission';
+import Nav from './components/Nav';
+import MobileNav from './components/MobileNav';
 
```

Comment:
I feel the search results should be wider, at least on large resolutions.
![image](https://user-images.githubusercontent.com/3792749/177590292-47f3ab09-a4af-49c5-8ad9-97b859ddaafc.png)

Other things I've noted:
- Seems like a lot of vertical whitespace between nav header and the heading "Project" with sauce logo next to it (I see this spacing was in the mockup, so it's just my opinion)
- Popular/Upvoted/Discussed/Recent/My Votes buttons do not appear to reflect state in a visual way (the way that list and grid layout icons do)
- Docs/Blog/Github/Youtube links have href="#" so they don't link anywhere

Reply:
One other update - I know it's also in the mockup but a lot of the gray text on white background doesn't have enough contrast for accessibility purposes. In particular, they are:
- the filter buttons at the top
- the paragraph descriptions of the repos
- the "Load More" large button
- The submit repo button (hard to believe, but it clocks in at 3.77 for contrast ratio with a target minimum of 4.5)

I suppose we could plan to offer a high contrast theme in the future.

Reply:
Hey @mtfoley, thanks for the constructive review! Gonna look into that. Also, a new design is coming up soon, hopefully, that will resolve some other issues 😊